extensions/amp-analytics/0.1/iframe-transport.js

```diff
@@ -166,6 +221,10 @@ export class IframeTransport {
     }
     ampDoc.body.removeChild(frameData.frame);
     delete IframeTransport.crossDomainIframes_[type];
+    if (this.longTaskObserver_) {
```

Comment:
I feel this won't work. 
For example we have IframeTransport 1, 2, 3. Only `IframeTransport2.longTaskObserver_ != null`
We call `#detach` in order 1, 2, 3.  In this way we will only try to disconnect longTaskObserver for `iframeTransport3`,  and fail to disconnect the real one.

Reply:
Ooh, good catch. You're absolutely right. Will fix.

Reply:
Done, though this moves us closer to a general refactor to only have 1 instance of PerformanceObserver. I am inclined to do that as a separate PR, but let me know if you disagree. Thanks!

Reply:
Yes, I think a separate following up PR is good enough : )